synapse/rest/client/v2_alpha/relations.py

```diff
@@ -153,6 +153,12 @@ def on_GET(self, request, room_id, parent_id, relation_type=None, event_type=Non
         from_token = parse_string(request, "from")
         to_token = parse_string(request, "to")
 
+        # Check if the event is redacted, and if so return an empty chunk
+        # list and zero tokens
+        if event.internal_metadata.is_redacted():
+            res = {"chunk": []}
```

Comment:
I'm struggling to understand the existing behaviour here (get_relations_for_event returns a pagination structure, which we then overwrite with different contents?), but it seems to return an `original_event`, which I can't find mentioned in the MSC. It's therefore hard to know if it is safe to not return it in this case.

I'm generally rather uneasy about unspecced behaviour which clients will come to rely on.

Reply:
The `original_event` came from https://github.com/matrix-org/synapse/pull/5626 which lead to a comment on the rewrite MSC [here](https://github.com/matrix-org/matrix-doc/pull/2154#pullrequestreview-258861183).

Reply:
After replacing things with `PaginationChunk`, `original_event` is now included in the response whether the event is redacted or not.

Not including it was an oversight on my part.